.circleci/workspace_status.sh

```diff
@@ -25,3 +28,15 @@ echo "GIT_SHA ${git_sha}"
 # Tag name, or GIT_SHA if not on a tag
 git_ref=$(git describe --tags --no-match --always --abbrev=40 --dirty | sed -E 's/^.*-g([0-9a-f]{40}-?.*)$/\1/')
 echo "GIT_REF ${git_ref}"
+
+# Plain git revision for linkstamping.
+echo "BUILD_SCM_REVISION" $(git rev-parse HEAD)
+
+# Git tree status for linkstamping.
+if [[ -n $(git status --porcelain) ]];
+then
+    tree_status="Modified"
```

Comment:
CI will apply the bazelrc first, this will always be modified when we build the container image form CI

Reply:
maybe check `$CI` or `$CIRCLE_CI` ?

Reply:
pass `.bazelrc` as a flag instead of relying on default?

Reply:
Could add composite file to .gitignore or put it in /tmp

Reply:
We decided to ignore the scm status for now.